lib/src/utils/parse/parse.dart

```diff
@@ -1,14 +1,13 @@
 import 'dart:convert';
 
-  /// Makes cmd arguments JSON parsable
+  /// Formats CLI arguments to make them JSON parsable.
   //  Turns [{'asdf':'asdf'}] -> [{"asdf":"asdf"}]
 parseArgsJson(String str) {
 //  Replace all single quotes ' with double quotes " so json.decode is able to read it. 
     String quoteReplace = str.replaceAll("\'", ('\"'));
-//  Json decode
+//  Json decode string
```

Comment:
U WERE SO CLOSE. . . . . . . . . . . . . (hint) .

Reply:
Lol hint please what should this be?

Reply:
oh punctuation jk